src/v/raft/recovery_rpc.json

```diff
@@ -0,0 +1,14 @@
+{
+    "namespace": "raft",
+    "service_name": "recovery_rpc",
+    "includes": [
+        "raft/recovery_rpc_types.h"
+    ],
+    "methods": [
+        {
+            "name": "reset_learner_state",
```

Comment:
can we use `remake` name here to minimise the number of names we use for the same operation ?